pootle/static/js/editor/mt/providers/CaighdeanTranslate.js

```diff
@@ -16,7 +16,7 @@ class CaighdeanTranslate extends MTProvider {
       apiKey,
       name: 'caighdean-translate',
       displayName: 'Caighdeán Translate',
-      url: 'http://borel.slu.edu/cgi-bin/seirbhis3.cgi',
+      url: '//borel.slu.edu/cgi-bin/seirbhis3.cgi',
       supportedLanguages: ['gd', 'gv', 'ga_IE', 'ga-IE'],
```

Comment:
Also right now this MT doesn't require specifying the target language since it always assumes `ga`, so my guess is that `ga` (or equivalent) is only necessary in `supportedLanguages` to enable this MT button on the translation editor UI.